src/otx/algorithms/detection/configs/detection/cspdarknet_yolox/data_pipeline.py

```diff
@@ -38,11 +38,27 @@
     dict(type="Pad", pad_to_square=True, pad_val=114.0),
     dict(type="Normalize", **__img_norm_cfg),
     dict(type="DefaultFormatBundle"),
-    dict(type="Collect", keys=["img", "gt_bboxes", "gt_labels"]),
+    dict(
```

Comment:
Why don't we using the `src/otx/algorithms/detection/configs/base/data/yolox_data_pipeline.py` for the base ?
Although most of things are different, using `base/data/yolox_data_pipeline.py` for this script like other scripts (`yolox_s`, `yolox_l`,...) looks good to me to keep the uniformity.

Reply:
Since yolox tiny is come from previous ote team and the other yolox variants come from mmdetection. Even though we can unify most of pipeline, normalization config should be different. And there is no clean way to change only normalization pipeline because we cannot override list.

Reply:
Ah, I missed the reply of Eunwoo's question, thanks.